Add design resources page and some pics (#411)
pxblue/doc-it: src/docs/design/intro.mdx

## Environment

We host and maintain our design assets through Figma. Figma is a powerful prototyping tool that facilitates collaboration and sharing of design assets between teams. It can be used on the web, and they also provide a desktop app for Mac and Windows. In order to use Figma, you will need to [sign up](https://www.figma.com/) for an account.

> ## What Materials are Available?

Reviewer: If it were me, I would keep this section here, but modify it to say something like:

"Blah blah blah, PX Blue has a number of materials available to help designers use the design system. These include our style guide (link to style guide landing page) and design patterns (link) as well as various Figma resources (link to resources page).

Author: I end up leaving the guideline CTAs there to make them more pronounced, since these are the items we'd like to emphasize.

## Environment

We host and maintain all [our design assets](/resources/designer) through Figma. Figma is a powerful prototyping tool that facilitates collaboration and sharing of design assets between teams. It can be used on the web, and they also provide a desktop app for Mac and Windows. In order to use Figma, you will need to [sign up](https://www.figma.com/) for an account.

## Guidelines

> PX Blue has a few design guidelines that serve as the foundation of the design system. They are intentional design choices we made to shape the PX Blue brand — what PX Blue applications should look like, and how PX Blue applications would interact with their users.

Reviewer: ```suggestion
PX Blue has design guidelines that serve as the foundation for the design system. These are intentional design choices we made to shape the PX Blue brand — what PX Blue applications should look like and how users should interact with them.
```
